src/components/Search/Search.js

```diff
@@ -14,21 +16,26 @@ import { useState, useEffect } from "react";
 export function Search({ getGifs, clearResults }) {
   const [inputValue, setInputValue] = useState("");
 
-  const handleChange = (event) => {
-    setInputValue(event.target.value);
+  const handleChange = (e) => {
+    setInputValue(e.target.value);
   };
 
-  const handleReset = (event) => {
-    event.preventDefault();
+  const handleReset = (e) => {
+    e.preventDefault();
     setInputValue("");
     clearResults();
   };
 
+  const handleSearch = (e) => {
+    e.preventDefault();
+    getGifs(inputValue);
+  };
+
   useEffect(() => {
-    const listener = (event) => {
-      if (event.code === "Enter" || event.code === "NumpadEnter") {
-        event.preventDefault();
-        getGifs(inputValue)
+    const listener = (e) => {
+      if (e.code === "Enter" || e.code === "NumpadEnter") {
```

Comment:
You should also check if `inputValue` has a length. Currently if the search input is empty and I press enter, the search network call will still be triggered.